packages/commonwealth/client/scripts/views/components/component_kit/cw_content_page.tsx

```diff
@@ -36,7 +36,7 @@ type ContentPageAttrs = {
   author?: m.Vnode;
   actions?: Array<MenuItem>;
   body?: m.Vnode;
-  comments?: Array<Comment<any>>;
+  comments?: m.Vnode;
```

Comment:
Reason for this change?

Reply:
It was left over from an earlier factoring.